packages/ws/README.md

```diff
@@ -44,6 +50,10 @@ const manager = new WebSocketManager({
 	rest,
 });
 
+manager.addListener(WebSocketShardEvents.Dispatch, (event) => {
```

Comment:
Not blocking, but shouldn't this be `.on` for consistency with our code?

Reply:
fixed